.ci/vfio_jenkins_job_build.sh

```diff
@@ -0,0 +1,276 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Run the .ci/jenkins_job_build.sh script in a VM
+# that supports VFIO, then run VFIO functional tests
+
+set -x
+
+cidir=$(dirname "$0")
+
+source /etc/os-release || source /usr/lib/os-release
+source "${cidir}/lib.sh"
+
+[ "${NAME}" == "Ubuntu" ] || die "Unsupported distro ${NAME}. Please use Ubuntu"
```

Comment:
Could you add a comment / GitHub issue URL explaining why this is Ubuntu-specific?